src/VS.Web.CG.Mvc/Identity/identitygeneratorfilesconfig.json

```diff
@@ -5,7 +5,9 @@
         "Name": "_LoginPartial",
         "SourcePath": "_LoginPartial.cshtml",
         "OutputPath": "Pages/Shared/_LoginPartial.cshtml",
-        "IsTemplate": true
+        "IsTemplate": true,
+        "AltPaths": [ "Pages/_LoginPartial.cshtml" ],
+        "ShouldOverWrite": "Never"
```

Comment:
If this file should never be overwritten, then it shouldn't be a candidate for scaffolding individually.

Reply:
@javiercn @mkArtakMSFT - thoughts on Prafull's comment?

Reply:
I'm not really an expert on the scaffolder, but sounds about right. We only want to generate the login partial if there isn't an existing one

Reply:
@javiercn @prafullbhosale @vijayrkn @mkArtakMSFT 
If we make _LoginPartial not be a candidate for scaffolding, then we'll also need to make a change in VS for the Identity Scaffolder UI - removing the checkbox for login partial.